apps/dev/pages/api/auth/[...nextauth].ts

```diff
@@ -4,7 +4,7 @@ import GitHubProvider from "next-auth/providers/github"
 import Auth0Provider from "next-auth/providers/auth0"
 import KeycloakProvider from "next-auth/providers/keycloak"
 import TwitterProvider, {
-  TwitterLegacy as TwitterLegacyProvider,
+  // TwitterLegacy as TwitterLegacyProvider,
```

Comment:
why is this commented out?

Reply:
this is an unused import that caused eslint warning, I guess this line can we removed instead